include/revng/Pipes/LinkForTranslation.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "revng/Pipeline/ContainerSet.h"
+#include "revng/Pipeline/Context.h"
+#include "revng/Pipeline/Contract.h"
+#include "revng/Pipeline/GenericLLVMPipe.h"
+#include "revng/Pipeline/LLVMGlobalKindBase.h"
+#include "revng/Pipeline/Target.h"
+#include "revng/Pipes/RevngPipes.h"
+#include "revng/Pipes/TemporaryFile.h"
+
+namespace pipeline {
+
+class LinkForTranslationPipe {
+public:
+  static constexpr auto Name = "Link for Translation Pipe";
+
+  std::array<ContractGroup, 1> getContract() const {
+    Contract BinaryPart(Binary,
+                        Exactness::Exact,
+                        0,
+                        Translated,
+                        2,
+                        InputPreservation::Preserve);
+    Contract ObjectPart(Object,
+                        Exactness::Exact,
+                        1,
+                        Translated,
+                        2,
+                        InputPreservation::Preserve);
+    return { ContractGroup({ BinaryPart, ObjectPart }) };
+  }
```

Comment:
Given a pipeline, I'd be very happy to dump a graph of the contracts.
But I guess this can wait.

Reply:
yeah, i guess it count as a new feature.